api/src/main/java/io/minio/StatObjectArgs.java

```diff
@@ -17,11 +17,28 @@
 package io.minio;
 
 /** Argument class of MinioClient.statObject(). */
-public class StatObjectArgs extends ObjectReadArgs {
+public class StatObjectArgs extends ObjectConditionalReadArgs {
+  protected StatObjectArgs() {}
+
+  public StatObjectArgs(ObjectConditionalReadArgs args) {
```

Comment:
As the name suggest  `StatObjectArgs` is an builder type then why is the constructor used. IMO it should be renamed to `StatObject` similar to `ComposeObject`.